### PR TITLE
Fixing example

### PR DESCRIPTION
**Pull request by amasiakiewicz**

### CODE REVIEW

File: examples/src/main/resources/application.properties [hunk 1]
```diff
@@ -1 +1,2 @@
-serviceDiscovery.address=http://localhost:8090/server/discovery
+serviceDiscovery.ratelServerAddress=http://localhost:8090/server/discovery
+serviceDiscovery.selfAddress=localhost:8080
```
Comment (slawekjaranowski) [inline examples/src/main/resources/application.properties]: Please insert new line

Comment (amasiakiewicz) [inline examples/src/main/resources/application.properties]: Yeah, just noticed that ravis has failed on checksyle. But locally when I do mvn test all is fine. How to run checksyle locally?

Comment (slawekjaranowski) [inline examples/src/main/resources/application.properties]: checkstyle plugin is bind to verify maven phase, so please run:
```mvn verify```

Comment (amasiakiewicz) [inline examples/src/main/resources/application.properties]: cool, thanks :-)

**Comment by amasiakiewicz** [comment]
Closing, changes will be in new one with additional fixes

**Pull request closed without merging**